app/views/list/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Signed Up Organisations</h1>
+  </div>
+  <%  @all_organisations.each do |organisation| %>
+    <%= render "list/table", organisation: organisation %>
```

Comment:
I'm confused. Why are we passing local variable `organisation` here? In `list/table` template we seem to be using only a global variable called `@all_organisation`, am I right?

If we use `@all_organisations.each` here, will it display N tables where N is the number of organisations we have? Is that we want? 🤔

Reply:
One table with N rows for each organisation.